src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background: black;
+  margin: 0;
+  padding: 0;
```

Comment:
body hasn't default paddings

Reply:
child elements don't inherite property in this case?

Reply:
nope, paddings and margins you specify only for element whit these was written for

Reply:
by the way you dont need to reset paddings in all elements